erts/emulator/nifs/win32/win_prim_file.c

```diff
@@ -898,9 +898,8 @@ posix_errno_t efile_read_handle_info(efile_data_t *d, efile_fileinfo_t *result)
         build_file_info_times(&native_file_info, result);
     } else if(is_path_root(path)) {
```

Comment:
This and the call to `efile_read_info` ought to take `&path` instead.

Reply:
Fixed, also for `is_path_root`.